ios/FluentUI/Vnext/Avatar/AvatarGroup.swift

```diff
@@ -0,0 +1,172 @@
+        let size: CGFloat = tokens.size.size
+        let x: CGFloat = size + tokens.interspace - tokens.ringOuterGap - tokens.ringThickness
+        let avatars: [AvatarView] = state.avatars
+        let maxDisplayedAvatars: Int = avatars.prefix(Int(state.maxDisplayedAvatars)).count
+        let overflowCount: Int = Int((avatars.count > maxDisplayedAvatars ? UInt(avatars.count - maxDisplayedAvatars) : 0) + state.overflowCount)
+        HStack(spacing: 0) {
+            ForEach(0 ..< maxDisplayedAvatars, id: \.self) { index in
+                let modify = tokens.style == .stack && (overflowCount > 0 || index + 1 < maxDisplayedAvatars)
+                state.avatars[index]
+                    .modifyIf(modify, { view in
+                        view.mask(AvatarCutout(xOrigin: x,
+                                               yOrigin: 0,
+                                               cutoutSize: size)
+                                    .fill(style: FillStyle(eoFill: true)))
+                    })
+                    .padding(.trailing, tokens.interspace)
+            }
+            if overflowCount > 0 {
+                createOverflow(count: overflowCount)
+            }
+        }
+    }
+
+    private func createOverflow(count: Int) -> AvatarView {
+        var avatar = AvatarView(style: .overflow, size: tokens.size)
+        let data = MSFAvatarStateImpl(style: .overflow, size: tokens.size)
+        data.primaryText = "\(count)"
+        data.image = nil
+        avatar.state = data
+        return avatar
+    }
+
+    /// Cutout shape for the succeeding Avatar in an Avatar Group in Stack style.
+    ///
+    /// xOrigin: beginning location of cutout on the x axis
+    ///
+    /// yOrigin: beginning location of cutout on the y axis
+    ///
+    /// cutoutSize: dimensions of cutout shape of the Avatar
+    private struct AvatarCutout: Shape {
+        var xOrigin: CGFloat
+        var yOrigin: CGFloat
+        var cutoutSize: CGFloat
+
+        func path(in rect: CGRect) -> Path {
+                var cutoutFrame = Rectangle().path(in: rect)
+                cutoutFrame.addPath(Circle().path(in: CGRect(x: xOrigin,
+                                                             y: yOrigin,
+                                                             width: cutoutSize,
+                                                             height: cutoutSize)))
+                return cutoutFrame
+        }
+    }
+}
+
+/// UIKit wrapper that exposes the SwiftUI AvatarGroup implementation
+@objc open class MSFAvatarGroup: NSObject, FluentUIWindowProvider {
+    @objc public init(style: MSFAvatarGroupStyle = .stack, size: MSFAvatarSize = .large, theme: FluentUIStyle? = nil) {
+        super.init()
+
+        avatarGroupView = AvatarGroup(style: style, size: size)
+        hostingController = UIHostingController(rootView: AnyView(avatarGroupView
+                                                                    .windowProvider(self)
```

Comment:
Have you tested the theme override per window?
Like... setting any token based on the theme override (theming demo controller) and see if it reflects on the control here